spec/schemas/_common.mapping.yaml

```diff
@@ -789,13 +789,14 @@ components:
         - type: object
           properties:
             value:
-              type: object
+              type: string
```

Comment:
Is this always a string? Can the value not be an integer, an object, etc?

Reply:
I have added the test to test integer in https://github.com/opensearch-project/OpenSearch/pull/14651.